app/controllers/meta_controller.rb

```diff
@@ -0,0 +1,23 @@
+class MetaController < ApplicationController
+  def sitemap
+    @urls = [root_url, high_voltage_urls, route_urls, service_specific_information_urls, tutorial_urls].flatten.uniq.sort
+  end
```

Comment:
This is fun… With [`flatten`](http://ruby-doc.org/core-2.4.2/Array.html#method-i-flatten), you can make an array of arrays a single array of values. I also added `uniq` and `sort` to the end.